tests/layer1/test_port_error.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import pytest
+import random
+import time
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.utilities import skip_release
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,  # disable automatic loganalyzer
+    pytest.mark.topology('any')
+]
+
+SUPPORTED_PLATFORMS = ["arista_7060x6"]
```

Comment:
@vdahiya12 Can you make this to PTP topology?

Reply:
fixed